### PR TITLE
Fix notifications

### PR DESCRIPTION
**Pull request by peacememories**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Affected Tests have been added/altered (for bug fixes / features)
- [ ] Docs have been reviewed and added/updated if needed (for bug fixes / features)
- [x] Build was run locally and `mvn install` succeeds

## Pull request type

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

- Fixes: Push notifications were not working correctly

## What is the new behavior?

- Push notification subscriptions are now properly sent to the server on each refresh

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by maxstolze** [comment]
@quasarchimaere is this PR already deprecated?

**Comment by quasarchimaere** [comment]
pretty much deprecated, but we should keep it open for now since we dont have a working pushNotification system so far, but we will probably overwrite the branch and manually add the changes made by @peacememories

File: webofneeds/won-owner-webapp/src/main/java/won/owner/web/rest/RestUserController.java [hunk 1]
```diff
@@ -311,30 +308,39 @@ public ResponseEntity transferUser(@RequestBody TransferUserPojo transferUserPoj
 
     @ResponseBody
     @RequestMapping(value = "/settings", produces = MediaType.APPLICATION_JSON_VALUE, method = RequestMethod.GET)
-    public UserSettingsPojo getUserSettings(@RequestParam("uri") String uri) {
+    public ArrayList<UserSettingsPojo> getUserSettings(@RequestParam(name = "uri", required = false) String uri) {
```
Comment (fkleedorfer) [inline webofneeds/won-owner-webapp/src/main/java/won/owner/web/rest/RestUserController.java:311]: Please use most generic type for the return type (ie, `List`)

File: webofneeds/won-owner-webapp/src/main/java/won/owner/web/websocket/WonWebSocketHandler.java [hunk 2]
```diff
@@ -391,6 +408,10 @@ private void notifyPerPush(final User user, final URI atomUri, final WonMessage
                     rootNode.put("atomUri", userAtom.getUri().toString());
                     rootNode.put("connectionUri", connectionUri.toString());
                     rootNode.put("icon", iconUrl);
+                    if (textMsg != null) {
+                        rootNode.put("message",
+                                        textMsg.length() < 50 ? textMsg : textMsg.substring(0, 46).concat(" ..."));
```
Comment (fkleedorfer) [inline webofneeds/won-owner-webapp/src/main/java/won/owner/web/websocket/WonWebSocketHandler.java:413]: Please use commons-lang's `StringUtils.abbreviate` function

**Pull request merged**